src/ScriptBuilder/ResourceReader.cs

```diff
@@ -8,7 +8,7 @@ static class ResourceReader
 
     public static string ReadResource(BuildSqlDialect sqlDialect, string prefix)
     {
-        var text = $"NServiceBus.Persistence.Sql.{prefix}_{sqlDialect}.sql";
+        var text = $"NServiceBus.Persistence.Sql.ScriptBuilder.{prefix}_{sqlDialect}.sql";
```

Comment:
Did all the resources move?

Reply:
The default namespace for the project, set via the RootNamespace project property, was set to `NServiceBus.Persistence.Sql` instead of `NServiceBus.Persistence.Sql.ScriptBuilder`, which is the actual namespace being used in the project, so I changed it the property to reflect the actual value.

It appears that the default namespace is used when embedding resources, so this needed to change as well to accommodate the namespace change.